### PR TITLE
Test BridgeStorageProvider class with the inclusion of the new federation type P2shErpFederation

### PR DESCRIPTION
**Pull request by nathanieliov**

### CODE REVIEW

File: rskj-core/src/test/java/co/rsk/peg/BridgeStorageProviderFederationTests.java [hunk 1]
```diff
@@ -0,0 +1,900 @@
...
+    private static final int ERP_FEDERATION_FORMAT_VERSION = 2000;
+    private static final int P2SH_ERP_FEDERATION_FORMAT_VERSION = 3000;
+
+    private final BridgeConstants bridgeConstantsRegtest = BridgeRegTestConstants.getInstance();
+    private final NetworkParameters btcRegTestParams = bridgeConstantsRegtest.getBtcParams();
+    private ActivationConfig.ForBlock activations;
+
+    @Before
+    public void setUp() throws Exception {
+
```
Comment (jeremy-then) [inline rskj-core/src/test/java/co/rsk/peg/BridgeStorageProviderFederationTests.java:58]: Is this `setUp` method needed if it's still empty?

**Review by jeremy-then — APPROVED** [review]
LGTM.

File: rskj-core/src/test/java/co/rsk/peg/BridgeStorageProviderFederationTests.java (hunk 1 above)
Comment (nathanieliov) [reply rskj-core/src/test/java/co/rsk/peg/BridgeStorageProviderFederationTests.java:58]: No, it's not needed. You right. I'll remove it.

**Comment by marcos-iov** [comment]
Moved to https://github.com/rsksmart/rskj/pull/1912

**Pull request closed without merging**